scipy/spatial/transform/_rigidtransformation.pyx

```diff
@@ -0,0 +1,1899 @@
+    notated B frames are next to each other and "cancel out".
+
+    .. parsed-literal::
+        :class: highlight-none
+
+                  ------------ B cancels out
+                  |         |
+                  v         v
+        p_A = t_A_B.apply(p_B)
+                ^
+                |
+                -------------- A is left
+
+    **Visualization**
+
+    >>> from scipy.spatial.transform import RigidTransformation as T
+    >>> from scipy.spatial.transform import Rotation as R
+    >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
+
+    The following function can be used to plot transformations with Matplotlib
+    by showing how they transform the standard x, y, z coordinate axes:
+
+    >>> colors = ("#FF6666", "#005533", "#1199EE")  # Colorblind-safe RGB
+    >>> def plot_transformed_axes(ax, t, name=None, scale=1):
+    ...     r = t.rotation
+    ...     d = t.translation
+    ...     loc = np.array([d, d])
+    ...     for i, (axis, c) in enumerate(zip((ax.xaxis, ax.yaxis, ax.zaxis),
+    ...                                       colors)):
+    ...         axlabel = axis.axis_name
+    ...         axis.set_label_text(axlabel)
+    ...         axis.label.set_color(c)
+    ...         axis.line.set_color(c)
+    ...         axis.set_tick_params(colors=c)
+    ...         line = np.zeros((2, 3))
+    ...         line[1, i] = scale
+    ...         line_rot = r.apply(line)
+    ...         line_plot = line_rot + loc
+    ...         ax.plot(line_plot[:, 0], line_plot[:, 1], line_plot[:, 2], c)
+    ...         text_loc = line[1]*1.2
+    ...         text_loc_rot = r.apply(text_loc)
+    ...         text_plot = text_loc_rot + d
+    ...         ax.text(*text_plot, axlabel.upper(), color=c,
+    ...                 va="center", ha="center")
+    ...     ax.text(*t.translation, name, color="k", va="center", ha="center",
+    ...             bbox={"fc": "w", "alpha": 0.8, "boxstyle": "circle"})
+
+    **Defining Frames**
+
+    Let's work through an example.
+
+    First, define the "world frame" A, also called the "base frame".
+    All frames are the identity transformation from their own perspective.
+
+    >>> t_A = T.identity()
+
+    We will visualize a new frame B in A's coordinate system. So we need to
+    define the transformation that converts coordinates from frame B to frame
+    A (A <- B).
+
+    Physically, let's imagine constructing B from A by:
+
+    1) Rotating A by +90 degrees around its x-axis.
+    2) Translating the rotated frame +2 units in A's -x direction.
+
+    From A's perspective, B is at [-2, 0, 0] and rotated +90 degrees about the
+    x-axis, which is exactly the transform A <- B.
+
+    >>> r_A_B = R.from_euler('xyz', [90, 0, 0], degrees=True)
+    >>> d_A_B = np.array([-2, 0, 0])
+    >>> t_A_B = T.from_rottrans(r_A_B, d_A_B)
+
+    ``t_B_A`` is the inverse, i.e. the transformation B <- A.
+
+    >>> t_B_A = t_A_B.inv()
```

Comment:
What is the idea behind introducing inverse transforms in this example?

It looks like you can convey the main point of the example without introducing `t_B_A` and `t_C_A`.

Reply:
Agree, moved the inversion down to where you suggested